Makefile

```diff
@@ -0,0 +1,106 @@
+# Makefile macros (or variables) are defined a little bit differently than traditional bash, keep in mind that in the Makefile there's top-level Makefile-only syntax, and everything else is bash script syntax.
+
+# .PHONY defines parts of the makefile that are not dependant on any specific file
+# This is most often used to store functions
+.PHONY: help clean build import tests
+
+# ======================
+# EXTERNAL VARIABLES
+# ======================
+
+REPOSITORY :=
+PREFIX :=
+TARGET := docker
+
+# ======================
+# INTERNAL VARIABLES
+# ======================
+
+_MAKE_DIR := .make_cache
+$(shell mkdir -p $(_MAKE_DIR))
+
+K8S_TAG := $(_MAKE_DIR)/.k8s_tag
+
+PLATFORM=amd64
```

Comment:
We should make this user definable and allow arm and riscv in future

Reply:
This should be user definable already. Variables defined in Makefile can be easily overridden, e.g. `make build PLATFORM=arm`, although now we miss the building logic when we change this. Anyhow, I have now also moved its position in the file and place it together with the other External variables.